src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,542 @@
+use Slim\Console\Command\Initializer\Dependency\OtherDependency;
+use Slim\Console\Command\Initializer\Dependency\PHPDIDependency;
+use Slim\Console\Command\Initializer\Dependency\PimpleDependency;
+use Slim\Console\Command\Initializer\Dependency\SlimPsr7Dependency;
+use Slim\Console\Command\Initializer\Profiles\AbstractInitProfile;
+use Slim\Console\Command\Initializer\Util\FileBuilder;
+use Slim\Console\Config\Config;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+use function array_keys;
+use function copy;
+use function file_get_contents;
+use function file_put_contents;
+use function get_class;
+use function getcwd;
+use function is_dir;
+use function is_file;
+use function mkdir;
+use function str_replace;
+use function touch;
+
+use const DIRECTORY_SEPARATOR;
+
+/**
+ * Init class implementation for profile Blank.
+ *
+ * @package Slim\Console\Command\Initializer\Profiles\blank
+ * @author Temuri Takalandze <me@abgeo.dev>
+ */
+class Init extends AbstractInitProfile
+{
+    /**
+     * @var bool
+     */
+    private $useDefaultSetup;
+
+    /**
+     * @var string
+     */
+    private $templatesDirectory;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function __construct(InputInterface $input, OutputInterface $output, ?Config $config = null)
+    {
+        parent::__construct($input, $output, $config);
+
+        $this->templatesDirectory = __DIR__ . DIRECTORY_SEPARATOR . 'templates';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function run(string $projectDirectory, bool $useDefaultSetup = false): int
+    {
+        $this->useDefaultSetup = $useDefaultSetup;
+        $exitCode = null;
+
+        if (0 !== ($exitCode = parent::run($projectDirectory, $useDefaultSetup))) {
+            return $exitCode;
+        }
+
+        if (0 !== ($exitCode = $this->createStructure($projectDirectory))) {
+            return $exitCode;
+        }
+
+        if (0 === ($exitCode = $this->setupDependencies($projectDirectory))) {
+            $this->io->success('New Slim project successfully created. Please run `composer install`.');
+        }
+
+        return $exitCode;
+    }
+
+    /**
+     * Create basic directory and files structure.
+     *
+     * @param string $projectDirectory
+     *
+     * @return int The Exit Code.
+     */
+    protected function createStructure(string $projectDirectory): int
+    {
+        $phpunitTemplate = null;
+        $directoryFullPath = getcwd() . DIRECTORY_SEPARATOR . $projectDirectory;
+        $composerJsonContent = $this->readComposerJson($directoryFullPath);
+        $directoriesToCreate = [
+            'bootstrap' => $this->config ? $this->config->getBootstrapDir() : 'app',
+            'index'     => $this->config ? $this->config->getIndexDir() : 'public',
+            'source'    => $this->config ? $this->config->getSourceDir() : 'src',
+            'logs'      => 'logs',
```

Comment:
Is this directory even needed, if the user decides not to install any logger?

Reply:
Currently, logger is the required component. @adriansuter, @l0gicgate  should we make it optional?

Reply:
@ABGEO it is in the skeleton but we should probably make it optional to give more flexibility

Reply:
@l0gicgate okay, should we also make other components optional?

Reply:
@ABGEO I don't think so for now. I know we're forcing people to use a container since we're not making it optional, but it shouldn't be. I don't know how you can soundly build an app without one.

Reply:
Yeap, I agree with you @l0gicgate